packages/dashboard/src/customization/widgets/lineSymbol/component.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { CSSProperties } from 'react';
+import { LineWidget } from '../types';
+import { SVG_STROKE_DASHED, SVG_STROKE_DOTTED, SVG_STROKE_SOLID } from '../constants';
+import { useWidgetActions } from '~/customization/hooks/useWidgetActions';
+import { DashboardWidget } from '~/types';
+import { LineAnchor } from './lineAnchor/component';
+import { useIsSelected } from '~/customization/hooks/useIsSelected';
+
+const LineWidgetComponent: React.FC<LineWidget> = (widget) => {
+  const { update } = useWidgetActions<DashboardWidget>();
+  const isSelected = useIsSelected(widget);
+
+  const fitFull = {
+    width: '100%',
+    height: '100%',
+  };
+
+  const {
+    lineStyle = 'solid',
+    color = 'black',
+    thickness = 5,
+    startX = 25,
+    startY = 25,
+    endX = 375,
+    endY = 375,
+  } = widget.properties;
+
+  let strokeString = SVG_STROKE_SOLID;
+
+  if (lineStyle === 'dashed') {
+    strokeString = SVG_STROKE_DASHED;
+  } else if (lineStyle === 'dotted') {
+    strokeString = SVG_STROKE_DOTTED;
+  }
+
+  const startDotStyle: CSSProperties = {
+    position: 'absolute',
+    left: `${startX}px`,
+    top: `${startY}px`,
+  };
+
+  const endDotStyle: CSSProperties = {
+    position: 'absolute',
+    left: `${endX}px`,
+    top: `${endY}px`,
+  };
+
+  return (
+    <div style={fitFull}>
```

Comment:
moved `fitFull`. i don't see the merit in moving `startDotStyle` and `endDotStyle` because we would only be able to move `position: 'absolute'`. let me know what you think.